xml/System.ServiceModel.Channels/MsmqTransportBindingElement.xml

```diff
@@ -100,7 +100,48 @@
         </remarks>
         <exception cref="T:System.ArgumentNullException">The context value is null.</exception>
         <exception cref="T:System.ArgumentException">The TChannel is not IOutputChannel or IOutputSessionChannel.</exception>
-        <exception cref="T:System.InvalidOperationException">MsmqTransportSecurity.MsmqProtectionLevel is EncryptAndSign and UseActiveDirectory is false.</exception>
+        <exception cref="T:System.InvalidOperationException">
+          ExactlyOnce is true and Durable is false.
+
+-or-
+
+UseActiveDirectory is true and QueueTransferProtocol is not Native.
+
+-or-
+
+CustomDeadLetterQueue is not null and DeadLetterQueue is not Custom.
+
+-or-
+
+DeadLetterQueue is set to Custom on Windows 2003/XP.
+
+-or-
+
+ExactlyOnce is false and DeadLetterQueue is Custom.
+
+-or-
+
+CustomDeadLetterQueue is set and if there is no write permission to write to queue or if queue does not exist.
```

Comment:
also for a future fix: ...and either there is no write permission to write to the queue, or the queue does not exist."